File: CucumberJVM/Chapter13/13.1-TheLocalScenarioContextPattern/Before/src/test/java/com/wimp/localscenariocontext/specs/stepdefinitions/AuthenticationStepDefinitions.java
package com.wimp.localscenariocontext.specs.stepdefinitions;

import com.wimp.localscenariocontext.app.services.AuthenticationService;
import io.cucumber.java.en.Given;

public class AuthenticationStepDefinitions {
    @Given("the customer {word} has logged in")
    public void theCustomerHasLoggedIn(String customerName) {
        AuthenticationService.login(customerName);
    }
}

File: Reqnroll/Chapter13/13.1-LocalScenarioContextPattern/Before/WIMP.Specs/StepDefinitions/AuthenticationStepDefinitions.cs
using Reqnroll;

using WIMP.App.Services;

namespace WIMP.Specs.StepDefinitions;

[Binding]
public class AuthenticationStepDefinitions
{
    [Given("the customer {word} has logged in")]
    public void GivenTheCustomerHasLoggedIn(string customerName)
    {
        AuthenticationService.Login(customerName);
    }
}
